detox/test/android/app/build.gradle

```diff
@@ -121,8 +126,15 @@ android {
 
 dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
+    compile project(":detox")
     compile "com.android.support:appcompat-v7:23.0.1"
     compile "com.facebook.react:react-native:+"  // From node_modules
+
+    testCompile 'junit:junit:4.12'
+    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
```

Comment:
do we need to `androidTestCompile` espresso here? I think we should just `androidTestCompile` detox instead, and remove line 129.